code/modules/mob/living/carbon/xenomorph/xenoupgrade.dm

```diff
@@ -95,6 +110,9 @@
 /mob/living/carbon/xenomorph/carrier/ancient
 	upgrade = XENO_UPGRADE_THREE
 
+//-----CARRIER-----//
+//================//
+//----HIVELORD----//
```

Comment:
CARRIER END
HIVELORD START 

Would be more readable, not just here but everywhere else